docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "116af785-a9b7-4eaf-8a86-b1eeb235da18",
+   "metadata": {},
+   "source": [
+    "# Fractional gates\n",
+    "\n",
+    "This page introduces two newly supported gate types on the IBM Quantum fleet of QPUs. These *fractional* gates are supported on [Heron QPUs](/guides/processor-types#heron) in the form of:\n",
+    "- $R_{ZZ}(\\theta)$ for $0 \\lt \\theta \\leq \\pi/2$\n",
+    "- $R_X(\\theta)$ for $0 \\lt \\theta \\leq 2\\pi$\n",
```

Comment:
There's a nuance here. If you call `rzz` with an angle outside the range `[0, pi/2]`, you'll get an error. However, if you call `rx` with an angle outside the range `[0, 2pi]`, then `rx` will be applied with `theta mod 2pi` and everything will be fine. I don't know why it's different for the two gates, I think @wshanks knows. Do we want to change here to
```
"- $R_X(\\theta)$ for any $\theta$\n"
```
I think it's OK also the way it is.

Reply:
That's right. `rx` should work for any theta. It doesn't need to be documented, but the reason for the differing angle ranges is that we already calibrate an `x` gate for each qubit which is `rx(pi)` and anything smaller just requires a smaller amplitude (and negative angles just require flipping the phase of the pulse). For `rzz`, we usually only calibrate a `cz` gate which is like `rzz(pi/2)`. In some cases we have the range to go to larger angles but not always, so pi/2 is what we can consistently commit to without changing the gate length.

Reply:
I ended up just using:

`$R_X(\theta)$ for any $\theta$`

Hopefully that works fine